python/examples/ingestion_with_python_config/telemetry_config.py

```diff
@@ -134,7 +134,7 @@ def nostromos_lv_426() -> TelemetryConfig:
             RuleConfig(
                 name="overvoltage",
                 namespace="voltage",
-                namespace_rules=rule_namespaces,
+                namespace_rules_dict=rule_namespaces,
```

Comment:
Sorry I wanted to rename this arg to be a little bit clearer and it blew up the diff a little bit

Reply:
Can we rename `namespace_rules_dict  to `namespaced_rules`?